### PR TITLE
Div fix ups for component library

### PR DESCRIPTION
**Pull request by amfestoy**
This PR is easier to read commit for commit

### CODE REVIEW

File: src/stories/Foundation/StatusTag/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  [key in ErrorSeverity]: { border: string; background: string; color: string };
+} = {
+  warning: { border: 'sunYellow', background: 'cheesecakeYellow', color: 'autumnYellow' },
+  error: { border: 'petitePink', background: 'whisperCoral', color: 'redWine' },
+  success: { border: 'springGreen', background: 'whisperGreen', color: 'oreganoGreen' },
+  neutral: { border: 'grey76', background: 'grey91', color: 'grey35' },
+  info: { border: 'plumbagoBlue', background: 'whisperBlue', color: 'royalBlue' },
+};
+
+const Wrapper = styled.div<{ severity: ErrorSeverity }>`
```
Comment (RostiMelk) [inline src/stories/Foundation/StatusTag/index.tsx:16]: Should maybe be a `p` or `span` so it stays inline? Seems a bit more semantic.

File: src/stories/Foundation/StatusTag/index.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  children: string;
+  severity: ErrorSeverity;
+}
+
+/**
+ * StatusTag component
+ */
+export const StatusTag = ({ className, children, severity }: StatusTagProps) => {
+  return (
+    <Wrapper severity={severity} className={className}>
```
Comment (RostiMelk) [inline src/stories/Foundation/StatusTag/index.tsx:37]: Make sure to include `role="status"` :)))

File: src/stories/Foundation/Text/index.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import styled from 'styled-components';
 
 export const Wrapper = styled.p<TextProps>`
   margin: 0;
-  line-height: 1.2em;
+  line-height: 1.6;
```
Comment (RostiMelk) [inline src/stories/Foundation/Text/index.tsx:7]: Great!!!

**Review by RostiMelk — COMMENTED** [review]
Looks great, just some minor comments 👏

File: src/stories/Foundation/StatusTag/index.tsx (hunk 2 above)
Comment (amfestoy) [reply src/stories/Foundation/StatusTag/index.tsx:38]: uuu nice!

File: src/stories/Foundation/StatusTag/index.tsx (hunk 1 above)
Comment (amfestoy) [reply src/stories/Foundation/StatusTag/index.tsx:16]: Agree!

**Pull request merged**